virttest/iscsi.py

```diff
@@ -237,6 +237,9 @@ def __init__(self, params, root_dir):
                 self.emulated_expect_size = block_size * size
                 self.create_cmd = ("dd if=/dev/zero of=%s count=%s bs=%sK"
                                    % (self.emulated_image, size, block_size))
+            else:
+                raise exceptions.TestError("Image size provided is not in valid"
```

Comment:
we can hint user with the param name aswell, `image_size`

Reply:
only after parsing `image_size` we error out if the units are not present. So user should already be aware of that param when he hits this error. Agree?

Reply:
Make sense..